modules/simple_forms_api/app/models/simple_forms_api/vba_21_0972.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  class VBA210972
+    include Virtus.model(nullify_blank: true)
+
+    attribute :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def metadata
+      {
+        'veteranFirstName' => @data.dig('veteran', 'full_name', 'first'),
+        'veteranLastName' => @data.dig('veteran', 'full_name', 'last'),
+        'fileNumber' => @data.dig('veteran', 'va_file_number').presence || @data.dig('veteran', 'ssn'),
+        'zipCode' => @data.dig('veteran', 'address', 'postal_code'),
```

Comment:
remember to update these as well with proper mappings!

Reply:
example what I see right now
```
{"veteranFirstName"=>nil, "veteranLastName"=>nil, "fileNumber"=>nil, "zipCode"=>nil, "source"=>"VA Platform Digital Forms", "docType"=>"21-0972", "businessLine"=>"CMP"}
```

Reply:
Where do you see `veteranFirstName`, etc.? The data returned from the FE is `{ veteran_full_name: { first: nil, last: nil } }`, right?

Reply:
Yes, that's correct. The `veteranFirstName` here is a required key in the downstream API spec, so we are mapping ours ({ veteran_full_name: { first:'first' } }) to theirs
See PUT `/path` in [Benefits Intake API docs](https://developer.va.gov/explore/api/benefits-intake/docs?version=current)